.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+lib
```

Comment:
if this line was taken out, Travis would automatically check in the build version when it ran `npm publish`, but I would prefer never to check in built files for various 'best practice' reasons.